SchemaDatabase/SGr/Generic/BaseTypes.xsd

```diff
@@ -454,199 +454,199 @@ https://github.com/SmartgridReady/SGrSpecifications  -->
+      <!-- <enumeration value="DEGREES_ANGULAR" /> -->
       <enumeration value="DEGREES_CELSIUS" />
-      <enumeration value="DEGREES_CELSIUS_PER_HOUR" />
-      <enumeration value="DEGREES_CELSIUS_PER_MINUTE" />
-      <enumeration value="DEGREES_FAHRENHEIT" />
-      <enumeration value="DEGREES_FAHRENHEIT_PER_HOUR" />
-      <enumeration value="DEGREES_FAHRENHEIT_PER_MINUTE" />
-      <enumeration value="DEGREES_KELVIN" />
-      <enumeration value="DEGREES_KELVIN_PER_HOUR" />
-      <enumeration value="DEGREES_KELVIN_PER_MINUTE" />
+      <!-- <enumeration value="DEGREES_CELSIUS_PER_HOUR" /> -->
+      <!-- <enumeration value="DEGREES_CELSIUS_PER_MINUTE" /> -->
+      <!-- <enumeration value="DEGREES_FAHRENHEIT" /> -->
+      <!-- <enumeration value="DEGREES_FAHRENHEIT_PER_HOUR" /> -->
+      <!-- <enumeration value="DEGREES_FAHRENHEIT_PER_MINUTE" /> -->
+      <!-- <enumeration value="DEGREES_KELVIN" /> -->
+      <!-- <enumeration value="DEGREES_KELVIN_PER_HOUR" /> -->
+      <!-- <enumeration value="DEGREES_KELVIN_PER_MINUTE" /> -->
       <enumeration value="DEGREES_PHASE" />
-      <enumeration value="DELTA_DEGREES_FAHRENHEIT" />
-      <enumeration value="DELTA_DEGREES_KELVIN" />
-      <enumeration value="FARADS" />
-      <enumeration value="FEET" />
-      <enumeration value="FEET_PER_MINUTE" />
-      <enumeration value="FEET_PER_SECOND" />
-      <enumeration value="FOOT_CANDLES" />
-      <enumeration value="GIGAJOULES" />
-      <enumeration value="GRAMS_OF_WATER_PER_KILOGRAM_DRY_AIR" />
-      <enumeration value="GRAMS_PER_MINUTE" />
-      <enumeration value="GRAMS_PER_SECOND" />
-      <enumeration value="HECTOPASCALS" />
-      <enumeration value="HENRYS" />
+      <!-- <enumeration value="DELTA_DEGREES_FAHRENHEIT" /> -->
+      <!-- <enumeration value="DELTA_DEGREES_KELVIN" /> -->
+      <!-- <enumeration value="FARADS" /> -->
+      <!-- <enumeration value="FEET" /> -->
+      <!-- <enumeration value="FEET_PER_MINUTE" /> -->
+      <!-- <enumeration value="FEET_PER_SECOND" /> -->
+      <!-- <enumeration value="FOOT_CANDLES" /> -->
+      <!-- <enumeration value="GIGAJOULES" /> -->
+      <!-- <enumeration value="GRAMS_OF_WATER_PER_KILOGRAM_DRY_AIR" /> -->
+      <!-- <enumeration value="GRAMS_PER_MINUTE" /> -->
+      <!-- <enumeration value="GRAMS_PER_SECOND" /> -->
+      <!-- <enumeration value="HECTOPASCALS" /> -->
+      <!-- <enumeration value="HENRYS" /> -->
       <enumeration value="HERTZ" />
-      <enumeration value="HORSEPOWER" />
-      <enumeration value="HOURS" />
-      <enumeration value="HUNDREDTHS_SECONDS" />
-      <enumeration value="IMPERIAL_GALLONS" />
-      <enumeration value="IMPERIAL_GALLONS_PER_MINUTE" />
-      <enumeration value="INCHES" />
-      <enumeration value="INCHES_OF_MERCURY" />
-      <enumeration value="INCHES_OF_WATER" />
-      <enumeration value="JOULE_SECONDS" />
+      <!-- <enumeration value="HORSEPOWER" /> -->
+      <!-- <enumeration value="HOURS" /> -->
```

Comment:
`HOURS` werden in einem FP (energy monitor) und verschiedenen Produkten (wärmepumpen) für Betriebsstunden benutzt. Sollen wir das in `MINUTES` ändern - das ist auch nicht sehr intuitiv.

Reply:
Wir können HOURS auskommentieren.